src/foam/java/refinements.js

```diff
@@ -552,6 +568,7 @@ foam.CLASS({
     ['javaType', 'int'],
     ['javaInfoType', 'foam.core.AbstractIntPropertyInfo'],
     ['javaJSONParser', 'new foam.lib.json.IntParser()'],
+    ['javaQueryParser', 'new foam.lib.json.IntParser()'],//'new foam.parse.IntParser()'
```

Comment:
After you add the expression to javaJSONParser then we can remove all these javaQueryParser statements when they are the same as the javaJSONParser.